3-probability/02-cond-prob-indep/ps.qmd

```diff
@@ -49,17 +49,6 @@ Are $A$ and $B$ independent?
 
 
 
-#### Question 4
-
-Suppose $A$ and $B$ are non-empty events such that $P(A) = 0.5$ and $P(B) = 0.7$.  What is the smallest and biggest that their union, $P(A \cup B)$, and their intersection, $P(A \cap B)$, can be?
```

Comment:
@shomurali Doesn't this occur in the previous problem set in a slightly different form? Was this deliberately duplicated?

Reply:
yes, sort of, since they still have a difficult time.